git-lfs/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: Brendan Forster <brendan@github.com>
+
+pkgname=('git-lfs')
+pkgver=1.1.2
+pkgrel=1
+pkgdesc="An open source Git extension for versioning large files"
+arch=('i686' 'x86_64')
```

Comment:
You want to build this with `mingw-makepkg`, right? Then you need to set `arch=('any')`. The reason is that 32-bit and 64-bit MinGW packages (as opposed to MSYS2 packages) can be installed side by side, in both a 32-bit and a 64-bit MSYS2 setup.

So as long as the package does not rely on the MSYS2 runtime (and Git LFS does not), we should use `mingw-makepkg` to generate the packages, and set `arch=('any')`.

To discern one from the other architecture, the `pkgname` needs to reflect the architecture: `pkgname=('${MINGW_PACKAGE_PREFIX}-git-lfs')` (compare with [the Git Credential Manager's `PKGBUILD`](https://github.com/git-for-windows/build-extra/blob/master/mingw-w64-git-credential-manager/PKGBUILD#L5)).